crypto/comp/c_zlib.c

```diff
@@ -86,7 +86,7 @@ static int zlib_stateful_expand_block(COMP_CTX *ctx, unsigned char *out,
                                       unsigned int olen, unsigned char *in,
                                       unsigned int ilen);
 
-/* memory allocations functions for zlib intialization */
+/* memory allocations functions for zlib initialization */
```

Comment:
see initialise vs initialize.  :)  don't bother changing all of them to one or the other.